Source/CrossingDetector/CrossingDetector.cpp

```diff
@@ -98,25 +99,20 @@ void CrossingDetector::createEventChannels()
     // event-related metadata!
     eventMetaDataDescriptors.clearQuick();
 
-    MetaDataDescriptor* eventLevelDesc = new MetaDataDescriptor(MetaDataDescriptor::FLOAT, 1, "Event level",
-        "Actual voltage level at sample where event occurred", "crossing.eventLevel");
-    chan->addEventMetaData(eventLevelDesc);
-    eventMetaDataDescriptors.add(eventLevelDesc);
+    MetaDataDescriptor* crossingLevelDesc = new MetaDataDescriptor(MetaDataDescriptor::FLOAT, 1, "Crossing level",
+        "Voltage level at first sample after crossing", "crossing.level");
+    chan->addEventMetaData(crossingLevelDesc);
+    eventMetaDataDescriptors.add(crossingLevelDesc);
 
     MetaDataDescriptor* threshDesc = new MetaDataDescriptor(MetaDataDescriptor::FLOAT, 1, "Threshold",
         "Monitored voltage threshold", "crossing.threshold");
     chan->addEventMetaData(threshDesc);
     eventMetaDataDescriptors.add(threshDesc);
 
-    MetaDataDescriptor* posOnDesc = new MetaDataDescriptor(MetaDataDescriptor::UINT8, 1, "Ascending on",
-        "Equals 1 if an event is triggered for ascending crossings", "crossing.positive");
-    chan->addEventMetaData(posOnDesc);
-    eventMetaDataDescriptors.add(posOnDesc);
-
-    MetaDataDescriptor* negOnDesc = new MetaDataDescriptor(MetaDataDescriptor::UINT8, 1, "Descending on",
-        "Equals 1 if an event is triggered for descending crossings", "crossing.negative");
-    chan->addEventMetaData(negOnDesc);
-    eventMetaDataDescriptors.add(negOnDesc);
+    MetaDataDescriptor* directionDesc = new MetaDataDescriptor(MetaDataDescriptor::UINT8, 1, "Direction",
+        "Direction of crossing: 1 = rising, 0 = falling", "crossing.direction");
+    chan->addEventMetaData(directionDesc);
+    eventMetaDataDescriptors.add(directionDesc);
```

Comment:
Replaced the `posOn` and `negOn` fields with `direction` to contain the direction of the crossing corresponding to each event (actually provides more information, in case both rising and falling are selected).